sdcm/nemesis.py

```diff
@@ -2398,6 +2399,163 @@ def disrupt_memory_stress(self):
         self.target_node.remoter.run(
             "stress-ng --vm-bytes $(awk '/MemTotal/{printf \"%d\\n\", $2 * 0.9;}' < /proc/meminfo)k --vm-keep -m 1 -t 100")
 
+    def disrupt_toggle_cdc_feature_properties_on_table(self):
+        """Manupulate cdc feature settings
+
+        randomly choose table form list and toggle cdc feature settings
+        if cdc was disaled, wait settings applying and scylla_cdc_log table
+        removing and enable cdc back
+        if cdc was enabled, run cdc stressor tool for table for period [5-20] minutes
```

Comment:
Change to:
- Find table with CDC enabled (skip nemesis if not).
- Get CDC settings.
- Randomly pick one of the settings and toggle/ if binary or change it if number.